src/Element/PluginSelect.php

```diff
@@ -127,6 +127,10 @@ public static function processPluginSelect(&$element, FormStateInterface $form_s
       $plugin = $plugin_manager->createInstance($target_plugin_id, $values['target_plugin_configuration']);
       if ($plugin instanceof PluginFormInterface) {
         $element['target_plugin_configuration'] = $plugin->buildConfigurationForm($element['target_plugin_configuration'], $form_state);
```

Comment:
this should move after #parents set to make ajax works on target plugin form (my typos)